src/DesignKit.tsx

```diff
@@ -2870,106 +2871,245 @@ const TooltipsSection: React.FC = () => {
   return (
-    <ComponentSection id="Columns" title="Columns">
-      <AntDCard>
+    <ComponentSection id="Column" title="Column and Row">
+      <AntDCard title="Column">
         <p>
-          The <code>{'<Columns>'}</code> component wraps child components to be displayed in
-          multiple columns.
+          A <code>{'<Column>'}</code> wraps child components to be displayed in a vertical column.
           <br />
-          The <code>{'<Column>'}</code> component can optionally be used to wrap the content for
-          each column and set its alignment.
         </p>
+        <hr />
+        <p>
+          The content within a Column can be aligned according to an <code>{'align'}</code> value.
+        </p>
+        <Row>
+          <Surface>
+            <Column align="left">Left-aligned column</Column>
+          </Surface>
+          <Surface>
+            <Column align="center">Center-aligned column</Column>
+          </Surface>
+          <Surface>
+            <Column align="right">Right-aligned column</Column>
+          </Surface>
+        </Row>
+        <hr />
+        <p>
+          A Column can have a vertical <code>{'gap'}</code>.
+        </p>
+        <p>
+          Column with <code>{'gap'}</code> set to 0:
+        </p>
+        <Column gap={0}>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+        </Column>
+        <p>
+          Column with <code>{'gap'}</code> set to 8 (default):
+        </p>
+        <Column gap={8}>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+        </Column>
+        <p>
+          Column with <code>{'gap'}</code> set to 16:
+        </p>
+        <Column gap={16}>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+          <Row>
+            <Surface>Row content</Surface>
+          </Row>
+        </Column>
+        <hr />
+        <p>
+          A Column can have a <code>{'width'}</code>, which is only applied when wrapped in a Row.
+        </p>
+        <p>Row with 3 Fill Width (default) columns:</p>
+        <Row>
+          <Column width="fill">
+            <Surface>Fill Width</Surface>
+          </Column>
+          <Column width="fill">
+            <Surface>Fill Width</Surface>
+          </Column>
+          <Column width="fill">
+            <Surface>Fill Width</Surface>
+          </Column>
+        </Row>
+        <p>Row with 3 Hug Width columns:</p>
+        <Row>
+          <Column width="hug">
+            <Surface>Hug Width</Surface>
+          </Column>
+          <Column width="hug">
+            <Surface>Hug Width</Surface>
+          </Column>
+          <Column width="hug">
+            <Surface>Hug Width</Surface>
+          </Column>
+        </Row>
+        <p>Row with 3 Fixed pixel width columns:</p>
```

Comment:
super nit: maybe change all the fixed pixel widths to something like 200px to make them more visually distinct from the 87px hug width containers at a glance